undefined

```diff
@@ -1,6 +1,4 @@
-console.log('Hello World');

```

Comment:
There is a syntax error. The opening script tag should not be present in this line.